staging/src/k8s.io/noderesourcetopology-api/manifests/crd.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     listKind: NodeResourceTopologyList
```

Comment:
it's not very clear what the state of the existing version of this CRD is... I can't comment on the line, but the `api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/1870` line above links to a PR which didn't look like it got API review. Is there a better link to an approved version of this API that matches the current checked-in state, or is this PR intended to be that review?

Reply:
As we have moved on with the project, the approach has evolved. Initial plan was to have an out-of tree exporter exposing per NUMA resource information as CRs per node consumed by in-tree scheduler plugin (which is the approach documented in https://github.com/kubernetes/enhancements/pull/1870). 

We later moved to both out-of-tree components relying on an external API as I mentioned above.

In addition to that , there is another KEP (https://github.com/kubernetes/enhancements/pull/2787) which proposes an API under k8s.io/api (like you suggested above) with kubelet populating the API and in-tree scheduler plugin consuming it to make topology aware placement decision. Since we have been focusing on enabling this feature and validating it, we haven't had the chance to pursue that due to capacity constraints. Do you think that would be a better way forward? 

Moving the API under kubernetes org so that it can be consumed from there in `node-feature-discovery` and `scheduler-plugins` rather than relying on the external API was [suggested](https://github.com/kubernetes/community/issues/6308#issuecomment-1011590184) as a good approach so it would nice for us enable that. I am open on `how` we want to do that :)